dask_jobqueue/core.py

```diff
@@ -110,15 +178,23 @@ def __init__(self,
 
         self.cluster = LocalCluster(n_workers=0, ip=host, **kwargs)
 
+        # plugin for tracking job status
+        self._scheduler_plugin = JobQueuePlugin()
+        self.cluster.scheduler.add_plugin(self._scheduler_plugin)
+        self.pending_jobs = self._scheduler_plugin.pending_jobs
+        self.running_jobs = self._scheduler_plugin.running_jobs
+        self.finished_jobs = self._scheduler_plugin.finished_jobs
+
+        # counter to keep track of how many jobs have been submitted
+        self._n = 0
```

Comment:
Do we still need to keep this counter?
I feel like we've got all the information we want in pending, running and finished jobs. We could even add some detailed status method wIth all that, maybe in another PR!